todo-app/src/App.js

```diff
@@ -1,13 +1,21 @@
-import React, { Component } from 'react';
+import React from 'react';
+import TodoList from './TodoList';
+import Todos from './Todos';
 
-class App extends Component {
+export default class App extends React.Component {
+  renderTodos(Todos) {
+    return Todos.map((todo, index) => {
+      return (
+        <TodoList key={index} text={todo.text} />
```

Comment:
Here you should have used a component that renders the text and checkbox in an `li` element. Now you are rendering for each todo item, one `ul` and one `li`, which results in the following

<img width="172" alt="screen shot 2017-10-28 at 11 16 36" src="https://user-images.githubusercontent.com/1057324/32133109-9df41e74-bbd1-11e7-9cf9-2ac8f57a3cef.png">

which is not the required result!